src/coreComponents/physicsSolvers/NonlinearSolverParameters.cpp

```diff
@@ -112,32 +112,37 @@ NonlinearSolverParameters::NonlinearSolverParameters( string const & name,
   registerWrapper( viewKeysStruct::timeStepDecreaseIterLimString(), &m_timeStepDecreaseIterLimit ).
     setApplyDefaultValue( 0.7 ).
     setInputFlag( InputFlags::OPTIONAL ).
-    setDescription( "Fraction of the max Newton iterations above which the solver asks for the time-step to be decreased for the next time step." );
+    setDescription( "Fraction of the max Newton iterations above which the solver asks for the time-step to be decreased for the next time-step." );
 
   registerWrapper( viewKeysStruct::timeStepIncreaseIterLimString(), &m_timeStepIncreaseIterLimit ).
     setApplyDefaultValue( 0.4 ).
     setInputFlag( InputFlags::OPTIONAL ).
-    setDescription( "Fraction of the max Newton iterations below which the solver asks for the time-step to be increased for the next time step." );
+    setDescription( "Fraction of the max Newton iterations below which the solver asks for the time-step to be increased for the next time-step." );
 
   registerWrapper( viewKeysStruct::timeStepDecreaseFactorString(), &m_timeStepDecreaseFactor ).
     setApplyDefaultValue( 0.5 ).
     setInputFlag( InputFlags::OPTIONAL ).
-    setDescription( "Factor by which the time step is decreased when the number of Newton iterations is large." );
+    setDescription( "Factor by which the time-step is decreased when the number of Newton iterations is large." );
 
   registerWrapper( viewKeysStruct::timeStepIncreaseFactorString(), &m_timeStepIncreaseFactor ).
     setApplyDefaultValue( 2.0 ).
     setInputFlag( InputFlags::OPTIONAL ).
-    setDescription( "Factor by which the time step is increased when the number of Newton iterations is small." );
+    setDescription( "Factor by which the time-step is increased when the number of Newton iterations is small." );
+
+  registerWrapper( viewKeysStruct::minTimeStepIncreaseIntervalString(), &m_minTimeStepIncreaseInterval ).
+    setApplyDefaultValue( 10 ).
+    setInputFlag( InputFlags::OPTIONAL ).
+    setDescription( "Value of the minimum interval, since the last time-step cut, for increasing the time-step." );
```

Comment:
what does "minimum interval" mean? It is hard to tell from this description. is it "minimum number of time steps"?

Reply:
Right, it's minimum number of time steps. Let me update the comment

Reply:
I find the comment below very clear:

```c++
GEOS_LOG_LEVEL_RANK_0( 1, GEOS_FMT( "{}: time-step size will be kept the same since it's been {} cycles since last cut.",
                                        getName(), m_numTimestepsSinceLastDtCut ) );
```

What about renaming `m_minNumCycleFixedTimeStepAfterCut`. It's a bit wordy but a probably clearer?

Reply:
Better `m_minFixedTimeStepNumCyclesAfterCut`

Reply:
I am fine with all options presented